jobs/seed_repos.groovy

```diff
@@ -0,0 +1,53 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+name = 'deis-seed-repos'
+repoName = 'seed-repo'
+
+job(name) {
+  description """
+    <p>Runs deis/seed-repo against all of the workflow components.</p>
+  """.stripIndent().trim()
+
+  scm {
+    git {
+      remote {
+        github("deis/${repoName}")
+      }
+      branch('master')
+    }
+  }
+
+  logRotator {
+    daysToKeep defaults.daysToKeep
+  }
+
+  triggers {
+    githubPush()
+  }
+
+  wrappers {
+    timeout {
+      absolute(defaults.testJob["timeoutMins"])
+      failBuild()
+    }
+    timestamps()
+    colorizeOutput 'xterm'
+    credentialsBinding {
+      string("GITHUB_ACCESS_TOKEN", "8e11254f-44f3-4ddd-bf98-2cabcb7434cd")
+    }
+  }
+
+  steps {
+    repos.each { Map repo ->
```

Comment:
This looks good -- the only problem I see (not at all part of this code, just sayin') is that we have a list of `repos` defined in `common.groovy` but which _may_ diverge from the repos that `seed-repo` tracks... I'd be inclined to use the list of repos (if exists) in `seed-repo` for iterating/seeding, if possible.